test/integration/test.sh

```diff
@@ -6,7 +6,7 @@ export CLUSTER_NAME="test-cluster"
 export REGISTRY_NAME="test-local-registry"
 
 export KIND_BINARY=./kind
-export KIND_URL=https://github.com/kubernetes-sigs/kind/releases/download/v0.9.0/kind-linux-amd64
+export KIND_URL=https://github.com/kubernetes-sigs/kind/releases/download/v0.11.1/kind-linux-amd64
```

Comment:
We really need move all kind dependencies into KTF since environment is configured from that repo.